tests/phpunit/elementor/modules/usage/test-module.php

```diff
@@ -439,8 +442,96 @@ public function test_dynamic_control() {
 		self::$document->save( $document );
 
 		// Check element without dynamic control.
-		$global_usage = get_option( Module::OPTION_NAME, [] );
+		$global_usage = get_option( Module::ELEMENTS_OPTION_NAME, [] );
 
 		$this->assertEquals( 0, count( $global_usage[ $doc_name ]['heading']['controls'] ) );
 	}
+
+	/**
+	 * TODO: This test will analyze only `$global_usage` of `wp-post' since in multisite kit is somehow shared,
+	 * And will be added to global usage.
+	 */
+	public function test_page_settings() {
+		$document_mock = self::$document_mock_default;
+
+		// Create document with controls
+		$first_document = $this->factory()->create_post();
+
+		$document_mock['settings'] = [
+			'background_background' => 'red',
+			'hide_title' => 'yes',
+		];
+
+		$first_document->save( $document_mock );
+
+		// Validate global usage.
+		$global_usage = get_option( Module::DOCUMENT_OPTION_NAME, [] );
+
+		$this->assertEqualSets( [
+			'background_background' => 1,
+			'hide_title' => 1,
+		], $global_usage['wp-post'] );
+
+		// Create document with the same controls
+		$second_document = $this->factory()->create_post();
+		$document_mock['settings'] = [
+			'background_background' => 'red',
+			'hide_title' => 'yes',
+		];
+
+		$second_document->save( $document_mock );
+
+		// Validate global usage increased.
+		$global_usage = get_option( Module::DOCUMENT_OPTION_NAME, [] );
+
+		$this->assertEqualSets( [
+			'background_background' => 2,
+			'hide_title' => 2,
+		], $global_usage['wp-post'] );
+
+		// Remove first document.
+		wp_delete_post( $first_document->get_main_id(), true );
+
+		$global_usage = get_option( Module::DOCUMENT_OPTION_NAME, [] );
+
+		// Validate global usage decreased.
+		$this->assertEqualSets( [
+			'template' => 1,
+			'background_background' => 1,
+		], $global_usage['wp-post'] );
+
+		// Remove second document.
+		wp_delete_post( $second_document->get_main_id(), true );
+
+		$global_usage = get_option( Module::DOCUMENT_OPTION_NAME, [] );
+
+		// Validate global usage decreased.
+		$this->assertArrayNotHaveKeys( ['wp-post'], $global_usage );
+	}
+
+	public function test_kit_settings() {
```

Comment:
Can you add
// Arrange
// Act
// Assert
comment?

and maybe split this test into 2 tests?

Reply:
Handled.